tests/cstor/volume/sclaeup_replica/scaleup_replica_utils_test.go

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
why can't we have these functions in operations.go ?

Reply:
IMO: These are tightly coupled with test cases so not pushed into operations.go.
`Reason1:`
operations.go can consist only of the functions which can be used by most of the tests and this helper functions (which are present in scaleup_replica_utils_test.go) is related to a particular test.
`Reason2:`
When we planned to run BDD in parallel then it should be lockless implementation.